kubernetes/manifests/cert-manager/cluster-issuer.yaml

```diff
@@ -8,11 +8,13 @@ metadata:
 
 spec:
   acme:
-    email: bradley.reynolds@darbia.dev
+    email: support@vipyrsec.com
```

Comment:
I don't even know if this email is live, but we need to rehost our email provider. Blocked for now; let me figure out a solution for this.